tests/passing/windows-preserves-crlf.t

```diff
@@ -0,0 +1,6 @@
+Running ocp-indent on windows should not rewrite CRLF to LF.
```

Comment:
You mean "should not rewrite LF to CRLF" ?

Reply:
Yes my bad!